sys-boot/refind/refind-0.11.4.ebuild

```diff
@@ -0,0 +1,197 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit toolchain-funcs flag-o-matic multiprocessing
+
+DESCRIPTION="The UEFI Boot Manager by Rod Smith"
+HOMEPAGE="https://www.rodsbooks.com/refind/"
+
+SRC_URI="mirror://sourceforge/project/${PN}/${PV}/${PN}-src-${PV}.tar.gz"
+
+LICENSE="BSD GPL-2 GPL-3 FDL-1.3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+FS_USE="btrfs +ext2 +ext4 hfs +iso9660 ntfs reiserfs"
+IUSE="${FS_USE} doc custom-cflags"
+
+DEPEND="|| ( >=sys-boot/gnu-efi-3.0.2 >=sys-boot/udk-2018-r1 )"
+
+DOCS="NEWS.txt README.txt docs/refind docs/Styles"
+
+UDK_ROOT="${EPREFIX}"/usr/share/udk
+[[ -d /usr/share/udk ]] && UDK_ROOT=/usr/share/udk
```

Comment:
Whatever you're trying to do here, it's definitely wrong. You aren't allowed to access filesystem in global scope.

Reply:
What is the suggested way to manage this? I need to find `udk` installation in order to compile rEFInd. But `udk` may have been installed with a prefix. The way I wanted to manage was to try first searching it using same prefix as rEFInd. If not found, I search it in its default place.
Do you think I should set this variable in `src_prepare` or do you think I should completely forget about prefix (and maybe display a warning in `udk` installation)?

Reply:
You should use EPREFIX unconditionally.